gradle-examples/gradle-android-library-example-ci-server/app/build.gradle

```diff
@@ -0,0 +1,47 @@
+apply plugin: 'com.android.library'
+
+android {
+    compileSdkVersion 30
+    buildToolsVersion "30.0.3"
+
+    defaultConfig {
+        applicationId "com.example.testlibrary"
```

Comment:
```suggestion
```

I'm not sure that a library can have an application ID: 
> com.android.builder.errors.EvalIssueException: Library projects cannot set applicationId. applicationId is set to 'com.example.testlibrary' in default config.

Reply:
You're correct! Only applications (not libraries) can set `applicationId`. My bad!